pkgs/by-name/bu/bun/hooks/default.nix

```
@@ -0,0 +1,86 @@
+{
+  lib,
+  stdenvNoCC,
+  jq,
+  moreutils,
+  cacert,
+  makeSetupHook,
+  bun,
+}:
+
+{
+  fetchDeps =
+    {
+      hash ? "",
+      pname,
+      workspaces ? [ ],
+      preInstall ? "",
+      installFlags ? [ ],
+      ...
+    }@args:
+    let
+      args' = builtins.removeAttrs args [
+        "hash"
+        "pname"
+      ];
+    in
+    stdenvNoCC.mkDerivation (
+      finalAttrs:
+      (
+        args'
+        // {
+          name = "${pname}-bun-deps";
+
+          nativeBuildInputs = [
+            cacert
+            jq
+            moreutils
+            bun
+          ] ++ args'.nativeBuildInputs;
```

```suggestion
          ] ++ args'.nativeBuildInputs or [ ];
```
This will fail by default, as most people will not pass this argument